galaxy_ng/app/api/v1/serializers.py

```diff
@@ -178,6 +181,12 @@ def get_avatar_url(self, obj):
         url = f'https://github.com/{username}.png'
         return url
 
+    def get_github_id(self, obj):
```

Comment:
You going to catch exception here because USerSocialAuth migration runs only for community profile https://github.com/ansible/galaxy_ng/blob/master/galaxy_ng/app/dynaconf_hooks.py#L197 

This query will fail for other profiles because the table doesnt exist.

Or you do EAFP (try/except) or LBYL (if "social_auth" in settings.INSTALLED_APPS)

Reply:
I didn't code for that because the entire v1 api isn't added to the router unless social auth is enabled. I'll add it anyway.

Reply:
So maybe the problem is at import time, it still errors not finding the table.

Reply:
Possibly, I'll look into it.